### ISSUE TITLE
Autocomplete accessibility issue

### ISSUE THREAD

**Issue by laugerud** [bring/hedwig#1612]
From Pia Wold Nilsen:
We have some accessibility issues with this component where the autocomplete suggestions are not accessible with keyboard.  We had a problem like this earlier with the dropdown component, where we fixed the accessibility issue by using the native dropdown, but for the autocomplete component we don’t have an option to use native. Is this something that will be fixed for the autocomplete component?
